configs/milvus.yaml

```diff
@@ -255,6 +255,10 @@ grpc:
     dialTimeout:      5000
     keepAliveTime:    10000
     keepAliveTimeout: 20000
+    maxMaxAttempts: 5
+    initialBackOff: 1.5
```

Comment:
backoff might be too small?
If back is 1.5 and retry is only 5, then backoff would be
1.5, 3, 6, 12, 24
My suggestion would be set  initialBackOff to 5
then retry is 5, 10, 20 , 40, 60